src/systems/a320_systems_wasm/src/lib.rs

```diff
@@ -45,12 +45,18 @@ fn create_aircraft_variable_reader(
     )?;
     reader.add("GEAR CENTER POSITION", "Percent", 0)?;
     reader.add("GEAR HANDLE POSITION", "Bool", 0)?;
+    reader.add("TURB ENG CORRECTED N1", "Percent", 1)?;
+    reader.add("TURB ENG CORRECTED N1", "Percent", 2)?;
```

Comment:
Depending on your needs, our custom engine model provides A32NX_ENGINE_N1:{index}, which are uncorrected values, and if I'm not mistaken would only give different behaviour from default model on startup and spooldown right now. But in the future it might also add windmilling of the engines in flight too, so I don't know it if would impact pressurization.

Reply:
> Depending on your needs, our custom engine model provides A32NX_ENGINE_N1:{index}, which are uncorrected values, and if I'm not mistaken would only give different behaviour from default model on startup and spooldown right now. But in the future it might also add windmilling of the engines in flight too, so I don't know it if would impact pressurization.

This is correct. First question would be to understand if CORRECTED engine parameters are needed for this feature or UNCORRECTED ones. There is a huge difference between both. Then, if UNCORRECTED N1 and N2 are the ones you really want, I suggest working with A32NX_ENGINE_N1/N2:index as explained above. If otherwise CORRECTED values are needed, I can create LVars for those.

Reply:
Thanks both for the info and suggestions. The engine N1 is purely needed to determine take off power, so if there's a better Simvar for that I can definitely change it!

The press system is not too bothered about engine power at any other stage (start up/spooldown/in-flight), purely take off, so that's why I took the CORRECTED parameter.